libs/types-common/src/Data/Misc.hs

```diff
@@ -277,12 +298,19 @@ instance Cql (Fingerprint a) where
   fromCql (CqlBlob b) = return (Fingerprint (toStrict b))
   fromCql _ = fail "Fingerprint: Expected CqlBlob"
 
+instance Arbitrary (Fingerprint Rsa) where
+  arbitrary =
+    pure $
+      Fingerprint
+        "\138\140\183\EM\226#\129\EOTl\161\183\246\DLE\161\142\220\239&\171\241h|\\GF\172\180O\129\DC1!\159"
+
 --------------------------------------------------------------------------------
 -- Password
 
 newtype PlainTextPassword = PlainTextPassword
   {fromPlainTextPassword :: Text}
-  deriving (Eq, ToJSON, Generic)
+  deriving stock (Eq, Generic)
+  deriving newtype (ToJSON)
```

Comment:
```suggestion
  deriving newtype (ToJSON)  -- FUTUREWORK: this should never be serialized
```

Reply:
I can try removing this instance. We won't get the roundtrip test, but I think in this case it's alright. If I run into trouble, I can write `toJSON = const (toJSON "hunter2")` and the same in the `Arbitrary` instance. What do you think?

Reply:
There is actually a case where we want to send this, when we generate an initial password on the server, see `NewProviderResponse'.

Reply:
Yes, that's why I added it as a FUTUREWORK.  :)

But now I'm thinking that we're going to use these types also for `servant-client`, so maybe nevermind.

Reply:
(I remember having had this discussion in the past, though, so maybe it makes sense to leave a comment that we do want to have ToJSON, and why.)